validator/client/runner.go

```diff
@@ -83,24 +83,24 @@ func run(ctx context.Context, v Validator) {
 				wg.Add(1)
 				go func(role pb.ValidatorRole, id string) {
 					defer wg.Done()
+					tpk := id
+					if len(id) > 16 {
```

Comment:
any reason to increase this to 16 bytes ?

Reply:
It's because tpk in this instance is a string (and this code was already in place beforehand, I just moved it).  When we move to keys of `[]byte` this will go away anyway.

Reply:
sounds good